### PR TITLE
Reorg for converters gelu (FX Converter Refactor [13/N]) <Target: converter_reorg_proto>

### PR DESCRIPTION
**Pull request by apbose**

### CODE REVIEW

**Comment by narendasan** [comment]
This PR needs to target converter_reorg_proto

File: py/torch_tensorrt/fx/converters/acc_ops_converters.py [hunk 1]
```diff
@@ -3403,33 +3403,14 @@ def acc_ops_gelu(
...
-        )
-
-    plugin_name = "CustomGeluPluginDynamic"
-    # type_id 0 for float32, 1 for  float16
-    type_id = trt.PluginField(
-        "type_id", np.array(0, dtype=np.int32), trt.PluginFieldType.INT32
+    return activation.gelu(
+        network,
+        target,
+        SourceIR.ATEN,
```
Comment (narendasan) [inline py/torch_tensorrt/fx/converters/acc_ops_converters.py:3409]: This is not the right IR

Comment (wushirong) [reply py/torch_tensorrt/fx/converters/acc_ops_converters.py:3409]: Can you update PR with your change?

Comment (apbose) [reply py/torch_tensorrt/fx/converters/acc_ops_converters.py]: Done

**Comment by narendasan** [comment]
Closing in favor of https://github.com/pytorch/TensorRT/pull/2070

**Pull request closed without merging**